### PR TITLE
Add test for integration test workflow

### PR DESCRIPTION
**Pull request by yanksyoon**
This PR adds support for integration test workflow testing to help securely make updates to the `integration_test.yaml` workflow.

Other workflows have been also changed to support `working-directory` input.

### CODE REVIEW

**Review by arturo-seijas — DISMISSED** [review]
LGTM

**Review by weiiwang01 — DISMISSED** [review]

**Review by jdkandersson — DISMISSED** [review]

**Review by jdkandersson — DISMISSED** [review]

**Review by arturo-seijas — DISMISSED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -24,6 +24,8 @@ The following workflows are available:
 | Name | Type | Default | Description |
 |--------------------|----------|--------------------|-------------------|
 | extra-arguments | string | "" | Additional arguments to pass to the integration test execution |
+| extra-test-matrix | string | '{}' | Additional test matrices to run the integration test combinations |
+| working-directory | string | "./" | Custom working directory for jobs to run on |
```
Comment (gregory-schiano) [inline README.md:28]: Could you sort it alphabetically ?

File: tests/workflows/integration/simple/simple.Dockerfile [hunk 2]
```diff
@@ -0,0 +1 @@
+FROM ubuntu:latest
```
Comment (gregory-schiano) [inline tests/workflows/integration/simple/simple.Dockerfile:1]: Could you set it to "22.04" instead ?

Comment (arturo-seijas) [reply tests/workflows/integration/simple/simple.Dockerfile:1]: This is already done in main :)

File: README.md (hunk 1 above)
Comment (yanksyoon) [reply README.md]: Sure! :D done!

**Review by gregory-schiano — APPROVED** [review]
LGTM

**Review by arturo-seijas — APPROVED** [review]
LGTM

**Pull request merged**